concepts/apply.md

```diff
@@ -0,0 +1,48 @@
+# Apply
+
+## Context
+
+Confidence has the concept of applying a flag, this means marking the flag as used by a user. From the Confidence documentation:
+
+`"When a flag has been resolved and the value has been used to configure the application, the flag is said to be applied"`
+
+When a flag is marked as applied, that application will populate the data that drives the metrics.
+
+## How to mark a flag as Applied
+
+A flag can be applied one of two ways.
+
+1. When the flag is requested from the backend
+2. When the flag is accessed in the code of an application.
+
+These are subtly different, depending on the use-case one is preferred over the other.
+
+## Backend Apply
+
+Applying on flag request introduces no additional work on the application. When the flag is requested from the
+Confidence Server it is marked as applied. This is regardless of if the value of the flag was used in the code.
+
+Backend apply is recommended for server applications, as we also recommend requesting a flag every time a flag is used.
+
+## Access Apply
```

Comment:
Should we just call it `Client Apply`? Or we could have the options being `backend | access`? I'm fine with both, but mixing the wording seems a bit confusing.

Reply:
i think access, but good shout